spec/models/project_spec.rb

```diff
@@ -51,7 +51,7 @@
       }
     }
     let(:nullable_properties) do
-      Set.new(%i[photos standard sdgs state latitude longitude tagline])
+      Set.new(%i[photos standard sdgs state latitude longitude])
```

Comment:
This type of unrelated change is why it would be nice to have those PRs auto-generated one day :D